Section 4 comments

lachellel opened the pull request:
57 comments reviewed
42 addressed with changes or no changes

Outstanding items: 
1.  lifetime and frequency for CRLs and OCSP - updates made as placeholders only; final will be discussed
2.  validity lifetime of identity evidence (#72)
3.  delegated ocsp responder certificates and renewals, and ocsp signing and responses in general



closes #217 closes #352 closes #353 closes #291 closes #283 closes #284 closes #354 closes #357 closes #285 closes #218 closes @219 closes #360 closes #286 partial #467 closes #362 closes #252 closes #253 closes #370 closes #371 closes #309 closes #372 closes #363 closes #225 closes #224 closes #364 closes #311 closes #220

lachellel committed:
rewrote 4.1.1 sentences to be cleaner; no change but closes #217

lachellel committed:
changes for 4.1.x and 4.2 closes #352 with no change, closes #354, closes #467 with no change, other issues open for discussion

lachellel committed:
section 4.3.x updates, closes #218

lachellel committed:
section 4.4 updates, closes #360, closes #286

lachellel committed:
section 4.5.1 placeholder for cert profiles partially addresses #467 comments

lachellel committed:
updates for 4.5.2 partial acceptance addresses #467 comments, 4.9.1.x closes #252 closes #253 update 4.9.2 closes #363 update 4.9.5 closes #224 update 4.9.6 closes #364

lachellel committed:
random updates, updates for #369, closing #370, closing #371

lachellel committed:
key compromise updates in 4.9.12; closes #309 closes #372

lachellel committed:
service availability section 4.10.2 updated; closes #291

lachellel committed:
updated 4.1.2 to clarify which key pair; closes #353

lachellel committed:
updated 3.2.2.8 CAA checks to remove the dns operator clause and section 4.2.1 related update

lachellel commented:
@konklone 

I just opened #488 - a PR from section5-comments branch to section4-comments branch.  Suggest we merge / accept that PR and all changes will flow to this branch and PR.

lachellel commented:
For section 6 updates, I'll edit against a copy of master so it's not out of sync!

konklone commented on certificate-policy.md:
```diff
@@ -434,82 +426,77 @@ All CAs shall specify the procedures for validating information and identity evi
-- At least one of the Certificate Transparency Logs shall be a CT Log operated by Google.
-- At least one of the Certificate Transparenty Logs shall be a CT Log not operated by Google.
+- At least one of the CT Logs shall be a log operated by Google.
+- At least one of the CT Logs shall be a Log operated by a government or business entity other than Google.
```
Nit: inconsistent capitalization of `log`/`Log` in these two lines.

konklone commented on certificate-policy.md:
```diff
@@ -669,26 +656,22 @@ In the latter case, the OCSP signing Certificate shall contain an extension of t
-
 ### 4.9.11 Other forms of revocation advertisements available
-If the Subscriber Certificate is for a high-traffic FQDN, the CA may rely on stapling, in accordance with [RFC4366], to distribute its OCSP responses. In this case, the CA shall ensure that the Subscriber "staples" the OCSP response for the Certificate in its TLS handshake. The CA shall enforce this requirement on the Subscriber either contractually, through the Subscriber Agreement or Terms of U ...
+If the Subscriber Certificate is for a high-traffic FQDN, the CA may rely on stapling, in accordance with RFC4366, to distribute its OCSP responses. In this case, the CA shall ensure that the Subscriber "staples" the OCSP response for the Certificate in its TLS handshake. The CA shall enforce this requirement on the Subscriber either contractually, through the Subscriber Agreement or Terms of Use ...
```
Is this compatible with root store requirements, that you could exempt OCSP responses being available by `GET` requests if you can ensure stapling?

lachellel replied:
@konklone The comment isn't a "nit"; your comment is valid.  

This policy statement is directly from the Baseline Requirements.  A related comment is in Issue #290 to "define a high-traffic FQDN".  I added your comments and mine to Issue #290.

konklone replied:
Thanks. And it was [my other comment](https://github.com/uspki/policies/pull/484#discussion_r158415034) that was the nit. :)

konklone reviewed (approved):
I added a nit comment, and asked a question. Neither should block the merge, so unless the question points to a significant issue, merge away. :+1:

lachellel committed:
updated for clarity

We agreed to limit server authentication certificates to 395 days - removing clause for certificates with lifetimes greater than 395 days (OBE). 
Updated for consistent capitalization and use of acronyms
Rearranged sentences
